scripts/jenkins/jobs-ibs/templates/cloud-mkcloud-job-crowbar_register-template.yaml

```diff
@@ -18,6 +18,7 @@
           predefined-parameters: |
             TESTHEAD=1
             cloudsource=develcloud{version}
+            want_sles12sp2=1
```

Comment:
@dirkmueller well, you could still add something like this to the template:
```
 project:
   name: cloud-mkcloud7-s390x
   version: 7
   previous_version: 6
   arch: s390x
   want-sles12sp2=1
   label: openstack-mkcloud-SLE12-{arch}
   jobs:
       - 'cloud-mkcloud{version}-job-crowbar_register-{arch}'
```
and change this line in the job definition to:
```
want_sles12sp2={want_sles12sp2}
```